src/components/MoneyRequestConfirmationList.js

```diff
@@ -273,6 +278,36 @@ function MoneyRequestConfirmationList(props) {
         );
     }, [confirm, props.selectedParticipants, props.bankAccountRoute, props.iouCurrencyCode, props.iouType, props.isReadOnly, props.policyID, selectedParticipants, splitOrRequestOptions]);
 
+    /**
+     * Grab the appropriate image URI based on file type
+     *
+     * @param {String} receiptPath
+     * @param {String} receiptSource
+     * @returns {*}
+     */
+    const getImageURI = (receiptPath, receiptSource) => {
+        const {fileExtension} = FileUtils.splitExtensionFromFileName(receiptSource);
+        const isReceiptImage = Str.isImage(props.receiptSource);
+
+        if (isReceiptImage) {
+            return receiptPath;
+        }
+
+        if (fileExtension === 'html') {
+            return ReceiptHTML;
+        }
+
+        if (fileExtension === 'doc' || fileExtension === 'docx') {
+            return ReceiptDoc;
+        }
+
+        if (fileExtension === 'svg') {
```

Comment:
If the receipt source is `svg`, I think we might be able to just render it like so:

```js
import { SvgUri } from 'react-native-svg';

...
...

<SvgUri
  width={CONST.RECEIPT.ICON_SIZE}
  height={CONST.RECEIPT.ICON_SIZE}
  uri={props.receiptSource}
/>
```

Reply:
I'm not sure if we should render SVGs for security reasons. We can inject XSS or malware in them

Reply:
oh my bad. ignore me. This is just a static svg in app

Reply:
I agree there is security concerns with rendering SVGs that maybe we should discuss before doing this

Reply:
I think we are protected from XSS-in-svg on web because our CSP prevents inline script tags from running.

However we don't currently have a CSP in the Electron app.